pkg/transaction/builder.go

```diff
@@ -246,6 +247,7 @@ func MakeContractDeploy(
 		0,
 		stacks.AnchorModeOnChainOnly,
 		stacks.PostConditionModeAllow,
```

Comment:
Since post conditions are added, they should be passed as an optional arg to builder transactions